flink-tests/src/test/java/org/apache/flink/test/state/operator/restore/AbstractOperatorRestoreTestBase.java

```diff
@@ -177,6 +177,9 @@ private String migrateJob(ClusterClient<?> clusterClient, Deadline deadline) thr
                 JobStatus.CANCELED,
                 jobCanceledFuture.get(deadline.timeLeft().toMillis(), TimeUnit.MILLISECONDS));
 
+        // TODO: the savepoint _metadata file for the current branch is stored in this
+        //  savepointPath, please create the corresponding test resource directory and
+        //  copy the _metadata file by hand.
```

Comment:
I'd suggest to make this change a separate hotfix commit. Additionally, we probably should move this to the JavaDoc of the class.